docs/targets/sendgrid.md

```diff
@@ -1,159 +1,71 @@
+After selecting the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
 
-* The SendgridTarget requires only one Secret, the APIKey, to be provided. 
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with a SendGrid event Target was successfully created.
 
-* Once its presences is confirmed, the Sendgrid Target Controller will create a Knative Service for the Adapter.  
+![Bridge status](../images/bridge-status-green.png)
 
-* A Status summary is added to the SendgridTarget object informing of the all conditions that the target needs.
+For more information about using SendGrid, please refer to the [SendGrid documentation][docs].
 
-* When ready the `status.address.url` will point to the internal point where Cloud Events should be sent.
+### Event Type
 
-### Sendgrid Target as an event Sink
+The SendGrid event target can consume events of any type[^s]
 
-#### Once deployed, Sendgrid Target Adapters are addressable. This means you can use it as a Sink for Knative components! Below you can find an example. 
+The SendGrid event Target accepts a [JSON][ce-jsonformat] payload with the following properties that will overwrite their respective `spec` parameters.[^s]
 
-* The included example trigger ['300-trigger.yaml'](https://github.com/triggermesh/knative-targets/blob/Sendgrid-merge-fix/sendgrid/samples/300-trigger.yaml) Is as follows. 
+| Name  |  Type |  Comment | Required
+|---|---|---|---|
+| **FromName** | string | Sender's name |false |
+| **FromEmail** | string | Sender's email | false |
+| **ToName** | string | Recipient's name | false |
+| **ToEmail** | string | Recipient's email | false |
+| **Message** | string | Contents of the message body | false |
 
-```yaml
-apiVersion: eventing.knative.dev/v1beta1
-kind: Trigger
-metadata:
-  name: sendgrid-sample-trigger
-spec:
-  broker: default
-  subscriber:
-    ref:
-      apiVersion: targets.triggermesh.io/v1alpha1
-      kind: SendgridTarget
-      name: triggermesh-email
-```
+If the **Message** key value is omitted within an event, the body of the email will contain the entire CloudEvent.
 
-* Once this, along with the other aformentioned requirements, have been deployed. It is now possible to implement something like the [awskinesis-sinkbinding.yaml](https://github.com/triggermesh/aws-event-sources/blob/master/config/samples/awskinesis-sinkbinding.yaml) example from the [aws-event-sources](https://github.com/triggermesh/aws-event-sources).
+### Example
 
-* By replacing the section starting at line [#18](https://github.com/triggermesh/aws-event-sources/blob/93df64d5b298cc73acfb93f835646cc1d06429c4/config/samples/awskinesis-sinkbinding.yaml#L18) with:
+The Target uses the `Event Source` to assign the `subject` of new emails in the following format:
 
+```go
+"Notification from " + string(event.Source()) + " Knative Event"
```

Comment:
Would @cab105  or @antoineco (or anyone) mind providing me with a better "front facing" format to display how the subject is formatted/propagated

Reply:
Just using ``` without the language specifier will work fine.

Reply:
While re-phrasing my question I came up with a way to write this that, i hope, is acceptable. 

Ref -> https://github.com/triggermesh/docs/blob/d8b6f0997c3776d5bc6539f267c901d1ced5d4e9/docs/targets/sendgrid.md#example